pyQuARC/code/checker.py

```diff
@@ -157,18 +158,18 @@ def _check_dependencies_validity(self, dependencies, field_dict):
                 return False
         return True
 
-    def _run_func(self, func, rule, rule_id, metadata_content, result_dict):
+    def _run_func(self, func, check, rule_id, metadata_content, result_dict):
```

Comment:
The terminology that we use:
rule: an entry in `rule_mapping.json`
check: an entry in `checks.json`

this was wrongly named `rule`